examples/wrapper/main.tf

```diff
@@ -0,0 +1,12 @@
+module "cloudtrail" {
+  source = "https://github.com/appzen-oss/terraform-aws-cloudtrail.git"
+
+  name                          = "dev-account-cloudtrail"
+  enable_log_file_validation    = true
+  include_global_service_events = true
+  insight_selector              = [ {insight_type = "ApiCallRateInsight"} ]
+  is_multi_region_trail         = false
```

Comment:
Why not multi-region?

Reply:
@snemetz This example is for public. I have already included code in our bitbucket tf-wrapper and is_multi_region_trail is true in our case. Refer https://bitbucket.org/appzeneng/tf-wrappers/src/master/environments/cloudtrail/